docs/who-can-apply.md

```diff
@@ -22,8 +22,8 @@ points.
 - Your website must be **publicly available**. We do not host search indices for
   websites that are available behind an authentication or a private network.
 
-- Your website must be a **documentation website**. We do not index blogs or
-  commercial content.
+- Your website must be a **technical documentation of an open source project or
+  a technical blog**. We do not index commercial content.
```

Comment:
this should link to the scraper I guess?

Reply:
It's already linked below so I thought it wasn't necessary:

```
DocSearch is entirely open source and you can run it yourself, or use any of our other API clients to take advantage of Algolia's features.
```